src/Resources/templates/generate_module_command/module/main.php.twig

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+* <your license here>
+*/
+if (!defined('_PS_VERSION_')) {
+    exit;
+}
+require_once _PS_MODULE_DIR_.'{{ module_name }}'.DIRECTORY_SEPARATOR.'vendor'.DIRECTORY_SEPARATOR.'autoload.php';
+class {{ module_name|capitalize }} extends Module
+{
+    protected $config_form = false;
+
+    public function __construct()
+    {
+        $this->name = '{{ module_name }}';
+        $this->version = '1.0.0';
+
+        $this->author = '??????'; //TODO: Edit
+        $this->need_instance = 0;
+
+        /**
```

Comment:
Can we remove this comment ? I think we don't care about this

Reply:
done